### ISSUE TITLE
excon shows WARNING for :debug param

### ISSUE THREAD

**Issue by tsabat** [leadmachineapp/simple_spark#6]
```
[excon][WARNING] Invalid Excon connection keys: :debug
/Users/timothysabat/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/excon-0.37.0/lib/excon/connection.rb:393:in `validate_params'
/Users/timothysabat/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/excon-0.37.0/lib/excon/connection.rb:57:in `initialize'
```

Looks like excon takes `debug_request` and `debug_response` params.

https://github.com/excon/excon/blob/master/README.md#options

**Comment by jakcharlton** [comment]
This looks like you're using a much older version of excon (0.37) - I am
currently on 0.49

Are you locked to the old version for another gem?

**Comment by jakcharlton** [comment]
And a quick look at Excon source seems to show that setting :debug works as
a 'set both' option ... not quite sure why they throw a warning for it

https://github.com/excon/excon/blob/f86765e4e5e81493e1cd3ff6d97a8c0c882be5e3/lib/excon/connection.rb#L67

**Comment by jakcharlton** [comment]
Closed by a969025

Bumped to version 1.0.1
